integration_tests/test_sync_views.py

```diff
@@ -0,0 +1,25 @@
+from integration_tests.helpers.http_methods_helpers import get, post
+
+
+def test_get_async_view(session):
+    r = get("/sync_view")
+    assert r.status_code == 200
```

Comment:
The status code is already tested in the `get` method and other http methods. You can remove this line in the added tests.

Reply:
Done!